src/main/java/org/jitsi/jigasi/transcription/VoskTranscriptionService.java

```diff
@@ -64,15 +66,32 @@
 
     private final static String EOF_MESSAGE = "{\"eof\" : 1}";
 
-    private final String websocketUrl;
+    private String websocketUrlConfig;
+
+    private String websocketUrl;
+
+    /**
+     * Assigns the websocketUrl to use to websocketUrl by reading websocketUrlConfig;
+     */
+    private void generateWebsocketUrl(Participant participant) throws ParseException {
```

Comment:
This { will fail the code style check.

Reply:
For some reason it didn't, but it's fixed now 👍

Reply:
Hum, then I need to revisit the config :)